### PR TITLE
v6.3 [new feature] responsiveAccordionTabs

### PR DESCRIPTION
**Pull request by designerno1**
#8651

@zurb/yetinauts 
maybe some native english speaker could take a look at the docs page english

### CODE REVIEW

File: docs/pages/responsive-accordion-tabs.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: Responsive Accordion Tabs
+description: Use the Markup of a Accordion or Tabs and create a Responsive Accordion Tabs.
```
Comment (colin-marshall) [inline docs/pages/responsive-accordion-tabs.md]: "Use the markup of the Accordion or Tabs components to create Responsive Accordion Tabs."

File: docs/pages/responsive-accordion-tabs.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+---
+
+## Basics
+
+Eighter <a href="accordion.html">Accordion Markup</a> or <a href="tabs.html">Tabs Markup</a> can be used for responsive switching these components
```
Comment (colin-marshall) [inline docs/pages/responsive-accordion-tabs.md]: "Either the Accordion markup or the Tabs markup can be used to responsively switch between the two components at different breakpoints."

Comment (andycochran) [inline docs/pages/responsive-accordion-tabs.md]: * Either

File: docs/pages/responsive-accordion-tabs.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    <div class="accordion-content" data-tab-content>
+      I would start in the open state, due to using the `is-active` state class.
+    </div>
+  </li>
+  <!-- ... -->
+</ul>
+```
+
+<div class="secondary callout">
+  <p>Once you put it all together, here's what you get!<br>Scale your browser down to see the toggle happen.<br>I am a `Accordion on small and large` and `Tabs on medium`</p>
```
Comment (colin-marshall) [inline docs/pages/responsive-accordion-tabs.md]: I am an Accordion on small and large, but I am Tabs on medium.

File: docs/pages/responsive-accordion-tabs.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    <p>Vivamus hendrerit arcu sed erat molestie vehicula. Sed auctor neque eu tellus rhoncus ut eleifend nibh porttitor. Ut in nulla enim. Phasellus molestie magna non est bibendum non venenatis nisl tempor. Suspendisse dictum feugiat nisl ut dapibus.</p>
+  </div>
+  <div class="tabs-panel" id="panel2">
+    <p>Suspendisse dictum feugiat nisl ut dapibus.  Vivamus hendrerit arcu sed erat molestie vehicula. Ut in nulla enim. Phasellus molestie magna non est bibendum non venenatis nisl tempor.  Sed auctor neque eu tellus rhoncus ut eleifend nibh porttitor.</p>
+  </div>
+</div>
+```
+
+<div class="secondary callout">
+  <p>Once you put it all together, here's what you get!<br>Scale your browser down to see the toggle happen.<br>I am a `Tabs on small and large` and `Accordion on medium`</p>
```
Comment (colin-marshall) [inline docs/pages/responsive-accordion-tabs.md]: I am Tabs on small and large and Accordion on medium.

**Comment by designerno1** [comment]
@colin-marshall 
just added 
`Accordion markup should have an ID, if no ID is present, an ID is autogenerated for switch to Tabs.`
good? ;-)

**Comment by colin-marshall** [comment]
First sentence:
The Accordion should have an `id` specified, but the plugin will automatically generate one if the `id` is omitted.

The second sentence I think I know what you're saying but am not positive. I will have to come back to this when I have a minute to dig into the functionality of this new component.

**Comment by rafibomb** [comment]
Damn - this is a hot feature! Let's get this across the line for 6.3.  @colin-marshall Great feedback! 

@designerno1 How can we support you on this?

**Comment by designerno1** [comment]
@rafibomb 
some real example testing could be fine, if there should be JS Unit Test it would be great if someone could support the feature there

**Comment by kball** [comment]
@Owlbertz can you take a dig into this, particularly with the unit testing in mind as you've been doing such good work on other unit tests?

**Comment by kball** [comment]
Given that all we are lacking at this point is testing, I'm going to add @colin-marshall's docs feedback and merge this to make it easier to test.

**Comment by kball** [comment]
Merged!

**Pull request merged**